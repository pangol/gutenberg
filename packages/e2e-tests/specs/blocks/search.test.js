/**
 * WordPress dependencies
 */
import {
	clickBlockAppender,
	getEditedPostContent,
	createNewPost,
} from '@wordpress/e2e-test-utils';

describe( 'Search', () => {
	beforeEach( async () => {
		await createNewPost();
	} );

	it( 'can be created by typing "/quote"', async () => {
		// Create a list with the slash block shortcut.
		await clickBlockAppender();
		await page.keyboard.type( '/search' );
		await page.keyboard.press( 'Enter' );

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );
