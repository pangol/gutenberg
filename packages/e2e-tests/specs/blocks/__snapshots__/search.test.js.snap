// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search can be created by typing "/quote" 1`] = `"<!-- wp:search /-->"`;
